Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -0,0 +1,4381 @@
+        if parsed_entries:
+            demisto.debug(f'Update the next entries: {parsed_entries}')
+        else:
+            demisto.debug('No new entries to update.')
+
+        return GetRemoteDataResponse(updated_incident, parsed_entries)
+
+    except Exception as e:
+        demisto.debug(f"Error in Jira incoming mirror for incident {parsed_args.remote_incident_id} \n"
+                      f"Error message: {str(e)}")
+
+        if "Rate limit exceeded" in str(e):
+            return_error("API rate limit")
+
+        if updated_incident:
+            updated_incident['in_mirror_error'] = str(e)
+        else:
+            incident_update = {
+                'id': parsed_args.remote_incident_id,
+                'in_mirror_error': str(e)
+            }
+        return GetRemoteDataResponse(
+            mirrored_object=incident_update,
+            entries=[]
+        )
+
+
+def get_updated_remote_data(client: JiraBaseClient, issue: Dict[str, Any], updated_incident: Dict[str, Any], issue_id: str,
+                            mirror_resolved_issue: bool, attachment_tag_from_jira: str, comment_tag_from_jira: str,
+                            user_timezone_name: str, incident_modified_date: datetime | None) -> List[Dict[str, Any]]:
+    """This function is in charge of returning the parsed entries of the updated incident, while updating
+    the content of updated_incident, which is in charge of holding the updated data of the incident (since arguments
+    are passed by reference, we can update the object in this function, and the changes to the object will be reflected
+    when we return to the function that called this function).
+
+    Args:
+        client (JiraBaseClient): The Jira client.
+        issue (Dict[str, Any]): The issue object returned from the API.
+        updated_incident (Dict[str, Any]): The object that will hold the updated data of the incident.
+        issue_modified_date (datetime): Timestamp of the last updated time of the issue in Jira.
+        incident_modified_date (datetime): Timestamp of the last updated time of the incident in XSOAR, with timezone equal
+        to the Jira user's timezone (using {'TIMEZONE': user_timezone_name} setting in dateparser.parse)
+        issue_id (str): The issue id
+        mirror_resolved_issue (bool): Whether to mirror Jira issues that have been resolved, or have the status `Done`.
+        attachment_tag (str): The attachment tag, to tag the mirrored attachments.
+        comment_tag (str): The comment tag, to tag the mirrored comments.
+
+    Returns:
+        List[Dict[str, Any]]:  Parsed entries of the updated incident, which will be supplied to the class GetRemoteDataResponse.
+    """
+    parsed_entries: List[Dict[str, Any]] = []
+    demisto.debug((f"\nUpdate incident:\n\tIncident name: Jira issue {issue.get('id')}\n\t"
+                  f"Reason: Issue modified in remote.\n"))
+    if mirror_resolved_issue:
```

Comment:
Looks good. 
Pease add a comment above every section here - #Close incidents of  resolved or Done issues
# Morroring attachments
#Mirroring comments